examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scroll_view/custom_scroll_view.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('What should be visible or not visible in the initial state.', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    expect(find.byType(Scaffold), findsOne);
+    expect(
+        find.descendant(
+          of: find.byType(Scaffold),
+          matching: find.byType(AppBar),
+        ),
+        findsOne);
```

Comment:
Super tiny whitespace nit (per the [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation)):

```dart
    // this would work
    expect(find.descendant(
      of: find.byType(Scaffold),
      matching: find.byType(AppBar),
    ), findsOne);


    // here's another option
    expect(
      find.descendant(
        of: find.byType(Scaffold),
        matching: find.byType(AppBar),
      ),
      findsOne,
    );
```

Reply:
It looks like these were changed to:

```dart
    expect(find.descendant(
          of: find.byType(Scaffold),
          matching: find.byType(AppBar),
        ),
        findsOne);
```

which is much better in terms of following the style guide! But it still doesn't quite match the whitespace conventions found elsewhere in the framework.